pkg/agent/agent.go

```diff
@@ -388,18 +388,35 @@ func (i *Initializer) initOpenFlowPipeline() error {
 }
 
 func (i *Initializer) FlowRestoreComplete() error {
-	// ovs-vswitchd is started with flow-restore-wait set to true for the following reasons:
-	// 1. It prevents packets from being mishandled by ovs-vswitchd in its default fashion,
-	//    which could affect existing connections' conntrack state and cause issues like #625.
-	// 2. It prevents ovs-vswitchd from flushing or expiring previously set datapath flows,
-	//    so existing connections can achieve 0 downtime during OVS restart.
-	// As a result, we remove the config here after restoring necessary flows.
-	klog.Info("Cleaning up flow-restore-wait config")
-	if err := i.ovsBridgeClient.DeleteOVSOtherConfig(map[string]interface{}{"flow-restore-wait": "true"}); err != nil {
-		return fmt.Errorf("error when cleaning up flow-restore-wait config: %v", err)
-	}
-	klog.Info("Cleaned up flow-restore-wait config")
-	return nil
+	// Issue #1600: A rare case has been found that the "flow-restore-wait" config was still true even though the delete
+	// call below was considered success. At the moment we don't know if it's a problem with OVSDB golang lib or OVSDB
```

Comment:
I feel unlikely a bug in OVSDB Golang lib, as the transaction code is simple and the same for all operations.

Reply:
I also took a look at the code and agree with you, but it's also hard to believe OVSDB will return success when no-op is done, so just list all components involved here.